src/compas_view2/app/controller.py

```diff
@@ -431,7 +431,7 @@ def view_front(self):
         None
 
         """
-        self.app.view.current = self.app.view.FRONT
+        self.app.view.current = self.app.view.VIEW_ANGLES["FRONT"]
```

Comment:
here it is a "viewangle", but in the config it is a "viewport". perhaps these names should be coordinated...

Reply:
certainly!